build.sbt

```diff
@@ -2,19 +2,38 @@ inThisBuild(
   List(
     scalaVersion := "2.12.3",
     organization := "org.scalameta",
-    version := "0.1-SNAPSHOT",
-    sources.in(Compile, doc) := Nil // faster publishLocal.
+    sources.in(Compile, doc) := Nil, // faster publishLocal
+    licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
+    homepage := Some(url("https://github.com/scalameta/language-server")),
+    developers := List(
+      Developer("olafurpg", "Ólafur Páll Geirsson", "olafurpg@gmail.com", url("https://geirsson.com")),
+      Developer("gabro", "Gabriele Petronella", "gabriele@buildo.io", url("https://github.com/gabro")),
+      Developer("laughedelic", "Alexey Alekhin", "laughedelic@gmail.com", url("https://github.com/laughedelic"))
+    ),
+    scmInfo in ThisBuild := Some(
+      ScmInfo(
+        url("https://github.com/scalameta/language-server"),
+        s"scm:git:git@github.com:scalameta/language-server.git"
+      )
+    ),
+    releaseEarlyWith := BintrayPublisher,
+    releaseEarlyEnableSyncToMaven := false,
+    releaseEarlyEnableLocalReleases := true, // TODO: remove once CI publishing is setup
+    publishMavenStyle := true,
+    bintrayOrganization := Some("scalameta"),
+    bintrayReleaseOnPublish := !isSnapshot.value
   )
 )
 
 lazy val noPublish = List(
-  publishArtifact in (Compile, packageDoc) := false,
-  publishArtifact in packageDoc := false,
-  sources in (Compile, doc) := Seq.empty,
```

Comment:
I'm not sure if this had some special meaning

Reply:
I think removing sources for the doc task makes it faster, that's why it was there

Reply:
Although it's already set in ThisBuild so it's probably redundant here

Reply:
I'd prefer to skip scaladoc and -sources.jar generation for publishLocal. Can we keep these only if `sys.env.contains("CI") == false`?

Reply:
If you're only planning to use publish on CI, this is the most sensible way of doing it. 👍

Reply:
> I'd prefer to skip scaladoc and -sources.jar generation for publishLocal. Can we keep these only if sys.env.contains("CI") == false?

But then it should be in `ThisBuild` not here, right? It's `noPublish` settings, that were used only in the test-worspace

Reply:
Yes, in this build works.

Reply:
Addressed in caadabd